auth0/src/test/java/com/auth0/android/request/internal/ThreadSwitcherShadow.java

```diff
@@ -16,10 +17,10 @@
 @SuppressWarnings({"unused", "RedundantSuppression"})
 public class ThreadSwitcherShadow {
 
-    private final RoboExecutorService executor;
+    private final InlineExecutorService executor;
```

Comment:
`RoboExecutorService` does not work with Mode.PAUSED. Updated to use `InlineExecutorService`, which behaves the same.